spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -3,7 +3,37 @@
 [partintro]
 --
 In this section you will learn how to configure Spring Cloud Data Flow server's features such as the relational database to use and security.
---
```

Comment:
Indeed, `-Pfull` build was failing on malformed docbook; Fixed after closing the `partintro`